### PR TITLE
Fix maxMessageSize validation in StandardMessageDispatcher

### PR DESCRIPTION
**Pull request by adarshan-intel**
maxDeviceMessageSize field accepts random integer values, should only accept 0 or the minimum packet size(300) bytes

### CODE REVIEW

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java [hunk 1]
```diff
@@ -709,6 +709,9 @@ protected void doHelloDevice(DispatchMessage request, DispatchMessage response)
     OnboardingConfig onboardConfig = new OnboardConfigSupplier().get();
     if (onboardConfig.getMaxMessageSize() != null) {
       int maxMessageSize = onboardConfig.getMaxMessageSize();
+      if (maxMessageSize != 0) {
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java:712]: What ?

Comment (adarshan-intel) [reply protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java]: Updated Code

Comment (adarshan-intel) [reply protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java]: https://github.com/fido-device-onboard/pri-fidoiot/pull/656/commits/78b527b5f220a516c163a589755246b026cfe1d4

**Comment by KiranSukhavasi** [comment]
closing this and deferring .

**Pull request closed without merging**